### ISSUE TITLE
DoChat can not use "Chat History Migration & Backup" feature

### ISSUE THREAD

**Issue by 643104191** [huan/docker-wechat#225]
Although I know it should be a network issue with Docker containers,
I'm still wondering if there are feasible solutions other than using a VPN to create a local network.

**Comment by huan** [comment]
That's a brilliant feature request!

I'm also interested in it.

**Comment by 643104191** [comment]
Is there any solution approach?
I recently learned about hooking in PC WeChat, but I’m not sure if it’s helpful for this issue.

**Comment by 643104191** [comment]
This issue has been resolved by using the `docker network create` command. The specific process is as follows:

1. By using commands like `netstat`, we discovered that when WeChat synchronizes chat records, it communicates by instructing the mobile phone to access the current WeChat runtime environment's IP address along with port `8000`.
2. However, the problem arises when containers do not specify network configurations. In such cases, the container's IP address defaults to `172.17.0.2/16`.
3. To address this, we created a network identical to the host machine's network and then exposed port `8000` using the `-p` flag.
